xla/service/gpu/tests/gpu_fused_mha_test.cc

```diff
@@ -1089,6 +1089,101 @@ class FlashAttentionBMMScalePaddingMaskSoftmaxBMM
   }
 };
 
+class FlashAttentionBMMScalePaddingMaskSoftmaxBMMF8
+    : public MultiHeadedAttentionTest {
+ protected:
+  void TestImpl_Flash_Attention_Inference_BMM1_NoMask_Softmax_BMM2_F8() {
+    if (skip_reason_) GTEST_SKIP() << *skip_reason_;
+    if (GetDnnVersionInfoOrDefault(backend().default_stream_executor()) <
+        se::dnn::VersionInfo(8, 9, 3)) {
+      GTEST_SKIP() << "Flash Attention requires cuDNN >= 8.9.3.";
```

Comment:
I think recently we switched to requiring 9.0.0. So I think you should update this here.